### ISSUE TITLE
Loading stuck

### ISSUE THREAD

**Issue by MagunRa** [AG-Guardian/KankaRelationExplorer#2]
I get the blue circle segment+ "loading xy", then the text disappears, and the circle segment spins forever (>10min) without anything showing or happening.

**Comment by AG-Guardian** [comment]
Sorry - I have stopped development of this app for now. The functionality of the app has been implemented into Kanka itself for boosted campaigns, so there is no longer any need to use it here.

**Comment by MagunRa** [comment]
I was sent from the kanka Discord to the app, if I need deeper relations.
I will have to write something for me self than.

**Comment by AG-Guardian** [comment]
Yeah sorry I dont have much free time to work on this project at the moment. I did have plans to come back and add a few features, like pushing relation changes to Kanka and some other things, but for the moment its on the back burner. If you want to make some changes I would be happy to review and merge your PRs
